bin/scripts/README.md

```diff
@@ -0,0 +1,83 @@
+# Utility Scripts for Candlepin
+
+## Refresh Data Extractor - refresh_data_extractor.sh
+
+### Purpose
+
+This script is for creating a facade in the locally running Candlepin that facilitates refreshes for an organization.  
+It typically gets used in conjunction with the org_migrator application so that orgs can be examined independently  
+of the running portal system.  
+
+### Usage
+
+`refresh_data_extractor.sh <input_file>`
+
+The input for this script is made by running an organization refresh in the portal with TRACE logging on. It is  
+usually compressed [.gz] , which the script is capable of handling.  
+The resulting file contains a JSON representation of the upstream data for the org. This script will parse the data  
+and establish an in-memory instance of the data that can be accessed during a local refresh. The instance will  
+remain until tomcat is restarted.
```

Comment:
I know this PR is merged, but can we clarify this bit because it is misleading, and also add more detailed instructions? The extractor script is only extracting the JSON file from the .log / .log.gz file, but it will not establish any in-memory data. The person using the script will need to take the resultant JSON, and import it into candlepin's 'HostedTestResource' endpoint by running: `curl -k -u admin:admin -X POST -F 'data=@upstream_data.json' -H 'Content-Type: multipart/form-data' https://<candlepin_hostname>:8443/candlepin/hostedtest/import/subscriptions` for the data to be created in memory